src/pudl/output/ferc1.py

```diff
@@ -177,7 +177,14 @@ def plants_small_ferc1(pudl_engine):
             on=["utility_id_ferc1", "plant_name_ferc1"],
             how="left",
         )
-        .assign(opex_nonfuel=lambda x: x.opex_total - x.opex_fuel)
+        .assign(
+            opex_nonfuel=lambda x: (
+                # This is because opex_total is not acutally the sum, need to add maint.
```

Comment:
ah! can we make a new issue to rename `opex_total` in the small plants table then?? it looks like the og column name is just 🥁 `operation`.

Reply:
ah ok that's fair

Reply:
oh wait, this would explain the weirdness I described in the comment above. How do you know the OG column name is just operation?

Reply:
done!